code-dot-org/code-dot-org
# set script_id when creating teacher feedback

FInishes [PLAT-683]. See [migrating teacher feedback ids](https://docs.google.com/document/d/1SJVYtKSmOchUOcGnAOWU4Ph1u2utOc896EX_acbAnJM/edit#) doc for more context. I also included some [notes](https://docs.google.com/document/d/1SJVYtKSmOchUOcGnAOWU4Ph1u2utOc896EX_acbAnJM/edit#heading=h.bmdh34k67b4c) showing the codepath on which teacher feedbacks are being created.

## Testing story

Manually verified as follows:
1. go to https://studio.code.org/s/csd3-2020/stage/14/puzzle/5 
2. select a section and a student, and leave feedback
3. in the chrome debugger, look at the script_id in the JSON response

Before this change, script_id is nil. after this change, script_id is set to a numerical value.

This change should have no user-facing impact, because it is setting data in the database which is never used. Therefore, I am relying on existing test coverage to confirm that this doesn't break anything.

# Reviewer Checklist:



[PLAT-683]: https://codedotorg.atlassian.net/browse/PLAT-683
